app/controllers/auth/sessions_controller.rb

```diff
@@ -35,9 +35,4 @@ def create
       flash[:notice] = nil
     end
   end
-
-  def destroy
-    super
-    flash[:notice] = nil
-  end
```

Comment:
All changes that remove hacks on devise controllers is very much appreciated. Thanks! :tada: